valhalla/baldr/traffictile.h

```diff
@@ -0,0 +1,128 @@
+#ifndef VALHALLA_BALDR_TRAFFICTILE_H_
+#define VALHALLA_BALDR_TRAFFICTILE_H_
+
+#include <cassert>
+#include <cstdint>
+#include <vector>
+
+namespace valhalla {
+namespace baldr {
+namespace traffic {
+struct Speed {
```

Comment:
Could we have some warning comments about any changes to the data layout has to be made with _great_ care.

Reply:
Would it be useful with a unittest to verify that it stays a POD? https://en.cppreference.com/w/cpp/types/is_pod

Reply:
we have tests like this for the regular tiles as well and yeah valhalla only increments major version when we make breaking changes to the structurs. this would be subject to the same thing